raiden/network/proxies/token_network.py

```diff
@@ -738,6 +711,8 @@ def set_total_deposit(
             ChannelBusyError: If the channel is busy with another operation
             RuntimeError: If the token address is empty.
         """
+        if not isinstance(total_deposit, int):
+            raise ValueError('total_deposit needs to be an integral number.')
```

Comment:
`s/integral/integer`

Reply:
It's the same thing. (https://www.quora.com/What-are-integral-numbers)

Reply:
I realize they're the same thing, i was meaning for using a simpler language,

Reply:
I will change this just for you. Simpler language is good for me.